databases/week2/School database_DB_w2.sql

```diff
@@ -0,0 +1,44 @@
+CREATE DATABASE school;
+USE school;
+
+-- Create a new database containing the following tables:
+	-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` VARCHAR (180) NOT NULL,
+`begins` DATETIME NOT NULL,
+`ends` DATETIME NOT NULL,
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+	-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE `student`(
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`email` varchar(255) NOT NULL,
+`phone` varchar(255) NULL,
+`class_id` int(10) unsigned NOT NULL,
+CONSTRAINT `fk_class_id` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- students
+INSERT INTO student ('id', 'name', 'email', 'phone', 'class_id') VALUES(1, 'Tiago Gonçalves', 'tiago86g@gmail.com', 42594575, 1);
```

Comment:
All the insert statements need to be changed
The column names may not be enclosed with `'`

INSERT INTO student (`id, name, email, phone, class_id`) VALUES(1, 'Tiago Gonçalves', 'tiago86g@gmail.com', 42594575, 1);

Reply:
Thank you Babak